features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -29,7 +29,7 @@ import Alert from 'AppComponents/Shared/Alert';
 import classNames from 'classnames';
 import CustomIcon from '../../Shared/CustomIcon';
 import LeftMenuItem from '../../Shared/LeftMenuItem';
-import { PageNotFound } from '../../Base/Errors/index';
```

Comment:
Shall we remove this file it's no longer in use ?

Reply:
Fixed